dashboard/app/helpers/levels_helper.rb

```diff
@@ -534,6 +534,10 @@ def blockly_options
       level_options[:sharedFunctions] = nil # TODO: handle non-standard pools
     end
 
+    if @level.is_a? Dancelab
+      level_options[:defaultSong] = @level.default_song
+    end
```

Comment:
I don't think we need this — the following ensures that all level properties are turned into options on `appOptions.level`:

https://github.com/code-dot-org/code-dot-org/blob/cda7db3f6fdcdc68781220d53735f40b3215b52d/dashboard/app/helpers/levels_helper.rb#L503-L505